docusaurus.config.js

```diff
@@ -17,77 +17,72 @@ const config = {
     [
       require.resolve('@docusaurus/plugin-client-redirects'),
       {
-        redirects: [
-        ]
-      }
-    ]
+        redirects: [],
+      },
+    ],
+    require.resolve('docusaurus-plugin-hotjar'), 
   ],
 
   presets: [
     [
-      "classic",
-      /** @type {import('@docusaurus/preset-classic').Options} */
-      ({
+      'classic',
+      {
         docs: {
-          sidebarPath: require.resolve("./sidebars.js"),
-          routeBasePath: "/",
+          sidebarPath: require.resolve('./sidebars.js'),
```

Comment:
why changing quotes from " to ' ?

Reply:
Honestly, because I was following the docs directly most of what I was looking at was having single quote (')

https://docusaurus.io/pt-BR/docs/2.3.1/api/docusaurus-config

I can revert back to double quote 🤷🏽

Reply:
revert back, no need to update quotes in this PR